Info.txt

```diff
@@ -294,4 +294,5 @@ Extended functions:
 .�	= pop a      push is int(a)
 .�	= pop a,(b)  push lcm(a, (b))
 .�	= pop a      insert a into global array and after quit, print array[input_1]
-.^	= pop a	     insert a into global array with immediate sorting and after quit, print array[input_1]
\ No newline at end of file
+.^	= pop a	     insert a into global array with immediate sorting and after quit, print array[input_1]
+.�  = pop a, b   surrounds a with b. If a is a number, it gets converted to a string. Works with strings and arrays.
```

Comment:
if you click the "view" link, it displays properly. Not sure why it's wrong here, but since the other symbols seem to do the same, I'm not too worried.